updater/lib/dependabot/update_files_command.rb

```diff
@@ -2,19 +2,43 @@
 
 require "base64"
 require "dependabot/base_command"
+require "dependabot/dependency_snapshot"
 require "dependabot/updater"
 
 module Dependabot
   class UpdateFilesCommand < BaseCommand
     def perform_job
+      # We expect the FileFetcherCommand to have been executed beforehand to place
+      # encoded files and commit information in the environment, so let's retrieve
+      # and decode them into an object.
+
+      # TODO: Parse the dependency files when instantiated
+      #
+      # We can pull the error handling for parser exceptions up into this class to
+      # completely remove the concern from Dependabot::Updater.
+      #
+      # This should happen separately to introducing the class as a shim.
+      #
+      # See: updater/lib/dependabot/dependency_snapshot.rb:52
```

Comment:
I think a no-op of sorts, probably in the shape of error forwarding, makes sense here.